tests/storage/hotplug.go

```diff
@@ -508,6 +508,8 @@ var _ = SIGDescribe("Hotplug", func() {
 			for _, volumeName := range dvNames {
 				By("removing volume " + volumeName + " from VM")
 				removeVolumeFunc(vm.Name, vm.Namespace, volumeName)
+			}
+			for _, volumeName := range dvNames {
```

Comment:
@awels Why did you move the `verifyVolumeNolongerAccessible` to a dedicated volume iteration?

Reply:
I encountered some issues where if I immediately tried to check if if the device was gone from inside the guest, it would prevent the disk from being removed. So instead of immediately checking, I decided to remove them all first and then check if they were gone. This would give enough time between removing and checking and it passes.